header/interface.go

```diff
@@ -63,9 +63,6 @@ var (
 	// ErrNotFound is returned when there is no requested header.
 	ErrNotFound = errors.New("header: not found")
 
-	// ErrNoHead is returned when Store is empty (does not contain any known header).
-	ErrNoHead = fmt.Errorf("header/store: no chain head")
-
 	// ErrNonAdjacent is returned when Store is appended with a header not adjacent to the stored head.
 	ErrNonAdjacent = fmt.Errorf("header/store: non-adjacent")
```

Comment:
this should also go in `header/store` pkg errors, no?

Reply:
no , because it is will lead to a cyclic dependence errors.

Reply:
ack